Can we not build latest LIBSPDM in Raspberry PI using mbedtls?

I ran these commands to build,

1. cmake -DARCH=arm -DTOOLCHAIN=GCC -DTARGET=Release -DCRYPTO=mbedtls ..
2. make copy_sample_key
3. make

Build output,
In file included from /home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/spdm_unit_test_common/spdm_unit_test.h:14,
                 from /home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/test_spdm_common/context_data.c:7:
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/test_spdm_common/context_data.c: In function ‘libspdm_test_secured_message_context_location_selection_case18’:
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/cmockalib/cmocka/include/cmocka.h:62:6: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]
     ((uintmax_t)(value))
      ^
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/cmockalib/cmocka/include/cmocka.h:1145:23: note: in expansion of macro ‘cast_to_uintmax_type’
     _assert_int_equal(cast_to_uintmax_type(a), \
                       ^~~~~~~~~~~~~~~~~~~~
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/test_spdm_common/context_data.c:1253:9: note: in expansion of macro ‘assert_int_equal’
         assert_int_equal(spdm_context->session_info[index].secured_message_context,
         ^~~~~~~~~~~~~~~~
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/cmockalib/cmocka/include/cmocka.h:62:6: error: cast from pointer to integer of different size [-Werror=pointer-to-int-cast]
     ((uintmax_t)(value))
      ^
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/cmockalib/cmocka/include/cmocka.h:1146:23: note: in expansion of macro ‘cast_to_uintmax_type’
                       cast_to_uintmax_type(b), \
                       ^~~~~~~~~~~~~~~~~~~~
/home/pi/Desktop/Hari/ResApp.bak20230403-1616/libspdm/unit_test/test_spdm_common/context_data.c:1253:9: note: in expansion of macro ‘assert_int_equal’
         assert_int_equal(spdm_context->session_info[index].secured_message_context,
         ^~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[2]: *** [unit_test/test_spdm_common/CMakeFiles/test_spdm_common.dir/build.make:76: unit_test/test_spdm_common/CMakeFiles/test_spdm_common.dir/context_data.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:5246: unit_test/test_spdm_common/CMakeFiles/test_spdm_common.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

--- a/unit_test/test_spdm_common/context_data.c
+++ b/unit_test/test_spdm_common/context_data.c
@@ -1257,7 +1257,7 @@ void libspdm_test_secured_message_context_location_selection_case18(void **state
     for (index = 0; index < LIBSPDM_MAX_SESSION_COUNT; index++)
     {
         /* Ensure the SPDM context points to the specified memory. */
-        assert_int_equal(spdm_context->session_info[index].secured_message_context,
+        assert_ptr_equal(spdm_context->session_info[index].secured_message_context,
                          secured_message_contexts[index]);
     }
 

